### ISSUE TITLE
log file request

### ISSUE THREAD

**Issue by detectiveli** [WXinlong/SOLO#181]
Dear contributor,

I tested your pre-trained SOLOV2... model that everything is perfect.

But when I train your code with the default settings of solov2_r101_dcn_fpn_8gpu_3x.py with one GPU (we do not have 8 GPUs), the loss gets stuck at some point.

To help me solve the above problems, may I ask you for the log files during the training process if it is convenient?

Kind Regards
